### PR TITLE
pr_22

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWATDZxn57qISpJ8hDXA###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_22@9
The return value of "length" must be used.

Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWATDZxo57qISpJ8hDXB###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_22@9
A "NullPointerException" could be thrown; "message" is nullable here.

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWATDZxn57qISpJ8hDW_###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_22@6
Complete the task associated to this TODO comment.

Comment (sns-seb) [reply src/main/java/org/sonarqube/A.java]: simulating user reply

Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java]: simulating bot reply

Comment (sns-seb) [reply src/main/java/org/sonarqube/A.java]: another user reply

**Review by sns-seb2 — CHANGES_REQUESTED** [review]
SonarQube has detected 2 issues

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb) [reply src/main/java/org/sonarqube/A.java:11]: my o my!

**Pull request closed without merging**